.github/workflows/build-on-release.yml

```diff
@@ -21,6 +21,10 @@ jobs:
             sm_branch: "1.11-dev"
             spcomp_version: "1.11.x"
 
+          - sm_version: "1.12"
```

Comment:
Ah right, so 1.11 is now considered oldstable &mdash; I don't think there's any compelling reason to keep 1.11 now, and the release steps weren't built with multiple SM versions in mind (there's only release archives for Linux and Windows, both containing 32-bit and 64-bit extension builds).

This PR may be effectively redundant with #22, but we'll see once that gets a final once-over and merged.